engines/wintermute/base/gfx/opengl/meshx.cpp

```diff
@@ -213,6 +213,33 @@ bool MeshX::update(FrameNode *parentFrame) {
 				}
 			}
 		}
+
+		for (uint i = 0; i < skinWeightsList.size(); ++i) {
+			finalBoneMatrices[i].transpose();
+			finalBoneMatrices[i].inverse();
+		}
+
+		for (uint32 i = 0; i < _vertexCount; ++i) {
+			for (int j = 0; j < 3; ++j) {
+				_vertexData[i * kVertexComponentCount + kNormalOffset + j] = 0.0f;
+			}
+		}
+
+		for (uint boneIndex = 0; boneIndex < skinWeightsList.size(); ++boneIndex) {
+			for (uint i = 0; i < skinWeightsList[boneIndex]._vertexIndices.size(); ++i) {
+				uint32 vertexIndex = skinWeightsList[boneIndex]._vertexIndices[i];
+				Math::Vector3d pos;
+				pos.setData(_vertexNormalData + vertexIndex * 3);
+				finalBoneMatrices[boneIndex].transform(&pos, true);
+				pos *= skinWeightsList[boneIndex]._vertexWeights[i];
+
+				for (uint j = 0; j < 3; ++j) {
+					_vertexData[vertexIndex * kVertexComponentCount + kNormalOffset + j] += pos.getData()[j];
+				}
+			}
+		}
+
+//		updateNormals();
```

Comment:
Commented out?

Reply:
Commented this out when I changed the normal update procedure. Just removed it now.